libs/openFrameworks/utils/ofUtils.cpp

```diff
@@ -469,10 +454,7 @@ bool ofToBool(const string& boolString) {
 	if(lower == "false") {
 		return false;
 	}
-	bool x = false;
-	istringstream cur(lower);
-	cur >> x;
-	return x;
+	return (bool)std::stoi(boolString);
```

Comment:
does this work? Presumably (haven't tried it), if `boolString` is `False`, then `stoi` will make this into some non-zero integer, which, when cast to bool, will be `True`, which is wrong. Is that correct?